### DISCUSSION TITLE
Can Mac sunshine boost into the project along with openssl and upnp?

### DISCUSSION THREAD

**Discussion by guochangxue** [LizardByte/community#445] [Sunshine] [answered]
Can Mac sunshine boost into the project along with openssl and upnp? How do you configure cmake?

**Comment by ReenigneArcher** [comment]
Your first question doesn't make any sense...

**Reply by guochangxue** [reply]
Hello, I want to compile sunshine on Mac so that it can run directly on other computers, how do I do this? Can be paid.
thank you

**Reply by ReenigneArcher** [reply]
We already provide macOS packages of Sunshine.

**Reply by guochangxue** [reply]
Thank you very much for your answer。
Below is which one is used on macOS packages？
![image001](https://github.com/LizardByte/Sunshine/assets/8470491/50c77a21-0e19-4c3f-9bd2-1ad8e7153dbe)

**Reply by ReenigneArcher** [reply, marked as answer]
Please check the installation documents. https://docs.lizardbyte.dev/projects/sunshine/en/latest/about/setup.html#install
